ambry-store/src/main/java/com.github.ambry.store/StoreMetrics.java

```diff
@@ -57,39 +57,39 @@
   private final MetricRegistry registry;
   private final String name;
 
-  public StoreMetrics(String name, MetricRegistry registry) {
+  public StoreMetrics(String replicaDir, MetricRegistry registry) {
     this.registry = registry;
-    this.name = name;
-    getResponse = registry.timer(MetricRegistry.name(BlobStore.class, name + "-storeGetResponse"));
-    putResponse = registry.timer(MetricRegistry.name(BlobStore.class, name + "-storePutResponse"));
-    deleteResponse = registry.timer(MetricRegistry.name(BlobStore.class, name + "-storeDeleteResponse"));
+    name = getPartitionName(replicaDir);
+    getResponse = registry.timer(MetricRegistry.name(BlobStore.class, name + ".StoreGetResponse"));
```

Comment:
instead of adding a "." in every single metric, why don't you add a "." at the end of `name`? I would prefer it that way.